listNetworkVRs.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+
+#      Copyright 2016, Leaseweb Technologies BV
+#
+#      Licensed to the Apache Software Foundation (ASF) under one
+#      or more contributor license agreements.  See the NOTICE file
+#      distributed with this work for additional information
+#      regarding copyright ownership.  The ASF licenses this file
+#      to you under the Apache License, Version 2.0 (the
+#      "License"); you may not use this file except in compliance
+#      with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#      Unless required by applicable law or agreed to in writing,
+#      software distributed under the License is distributed on an
+#      "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#      KIND, either express or implied.  See the License for the
+#      specific language governing permissions and limitations
+#      under the License.
+
+# Script to list and cycle all networks
+# Nuno Tavares - n.tavares@tech.leaseweb.com
+
+import time
+import sys
+import getopt
+from cloudstackops import cloudstackops
+import os.path
+from random import choice
+from prettytable import PrettyTable
+
+# Function to handle our arguments
+
+
+def handleArguments(argv):
+    global DEBUG
+    DEBUG = 0
+    global DRYRUN
+    DRYRUN = 1
+    global domainname
+    domainname = ''
+    global configProfileName
+    configProfileName = ''
+    global command
+    command = 'list'
+    global opFilter
+    opFilter = None
+    global opFilterNoRR
+    opFilterNoRR = None
+
+    # Usage message
+    help = "Usage: ./" + os.path.basename(__file__) + ' [options] ' + \
+        '\n  --config-profile -c <profilename>\t\tSpecify the CloudMonkey profile name to get the credentials from (or specify in ./config file)' + \
+        '\n  --debug\t\t\t\t\tEnable debug mode' + \
+        '\n  --restart -r\t\t\t\t\tRestarts network w/ cleanup=True' + \
+        '\n    --type <networkType> \t\t\tApplies filter to operation, Possible networkTypes: Isolated,Shared,VPC,Isolated' + \
+        '\n    --onlyNoRR \t\t\t\t\Selects only non-redudant VR networks' + \
```

Comment:
One tab is missing a 't' here. Same on line 59 below.

![screen shot 2016-03-13 at 18 18 58 pm](https://cloud.githubusercontent.com/assets/1630096/13730165/23ffb9f0-e948-11e5-9c80-27d58a80b3be.png)